### PR TITLE
[enhance] Parameterize Node Affinity

### PR DESCRIPTION
**Pull request by eduardogmisiuk**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Adds a variable to control node affinity. It is useful in tests where we must enforce the services to be in certain nodes to increase performance/prevent other services from disturbing the performance of other ones.

* **What is the current behavior?** (You can also link to an open issue here)
There is no node affinity mapping.

* **What is the new behavior (if this is a feature change)?**
The node affinity can be enabled or disabled via a parameter. The label in which each service will go can be modified too.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: roles/apigw/templates/apigw_routes_config_map.yaml.j2 [hunk 1]
```diff
@@ -114,8 +114,8 @@ data:
         curl  -s  -sS -X POST \
             --url ${kong}/services/auth-service/plugins/ \
             --data "name=rate-limiting" \
-            --data "config.minute=5" \
-            --data "config.hour=40" \
+            --data "config.minute=500" \
```
Comment (mprevide) [inline roles/apigw/templates/apigw_routes_config_map.yaml.j2:117]: I think that perhaps these values should not be changed.

File: roles/vernemq/templates/z_vernemq_cluster.yaml.j2 [hunk 2]
```diff
@@ -6,7 +6,8 @@ metadata:
   name: k8s
   namespace: {{ dojot_namespace }}
 spec:
-  baseImage: dojot/vernemq-dojot
+  #baseImage: dojot/vernemq-dojot
+  baseImage: jonaphael/vernemq
```
Comment (mprevide) [inline roles/vernemq/templates/z_vernemq_cluster.yaml.j2:10]: I think that perhaps these values should not be changed.

File: roles/vernemq/templates/z_vernemq_cluster.yaml.j2 [hunk 3]
```diff
@@ -44,6 +47,11 @@ spec:
...
     plugins: []
   serviceAccountName: vernemq-k8s
+{% if dojot_enable_node_affinity %}
+  nodeSelector:
+    dojot.components/group: "{{ dojot_node_vernemq_label }}"
+{% endif %}
   size: {{ dojot_vernemq_replicas }}
-  tag: {{ dojot_verne_version }}
+  #tag: {{ dojot_verne_version }}
+  tag: crl9
```
Comment (mprevide) [inline roles/vernemq/templates/z_vernemq_cluster.yaml.j2:56]: I think that perhaps these values should not be changed.

File: roles/x509-identity-mgmt/templates/x509_identity_mgmt_deployment.yaml.j2 [hunk 4]
```diff
@@ -48,6 +48,25 @@ spec:
           value: "production"
         - name: MONGO_URI
           value: "mongodb://mongodb:27017/x509-identity-mgmt"
+        # - name: EJBCA_DELTA_CRL_PERIOD_DEVICES_CA
```
Comment (mprevide) [inline roles/x509-identity-mgmt/templates/x509_identity_mgmt_deployment.yaml.j2:51]: Should this block be commenting?

**Comment by mprevide** [comment]
this seems like a great improvement, I just left some comments

File: roles/x509-identity-mgmt/templates/x509_identity_mgmt_deployment.yaml.j2 (hunk 4 above)
Comment (eduardogmisiuk) [reply roles/x509-identity-mgmt/templates/x509_identity_mgmt_deployment.yaml.j2]: It should not exist at all haha

**Pull request merged**